webpack.config.js

```diff
@@ -33,7 +33,10 @@ const babelrc = {
     'react',
     ['env', {targets, modules: false}],
   ],
-  plugins: ['syntax-dynamic-import'],
+  plugins: [
+    'syntax-dynamic-import',
+    'syntax-object-rest-spread',
```

Comment:
Strangely, I saw this in the yarn.lock but I got an error until I enabled it here. Is object spread the preferred paradigm for partial state updates?